docs/articles_en/openvino-workflow/model-optimization.rst

```diff
@@ -13,57 +10,103 @@ Model Optimization Guide
    model-optimization-guide/weight-compression
 
 
-Model optimization is an optional offline step of improving the final model performance
-and reducing the model size by applying special optimization methods, such as 8-bit
-quantization, pruning, etc. OpenVINO offers three optimization paths implemented in
-`Neural Network Compression Framework (NNCF) <https://github.com/openvinotoolkit/nncf>`__:
+Model optimization means altering the model itself to improve its performance.
+It is an optional step, typically used only at the development stage, so that the pre-optimized
+model is used in the final AI application.
 
-- :doc:`Post-training Quantization <model-optimization-guide/quantizing-models-post-training>`
-  is designed to optimize the inference of deep learning models by applying the
-  post-training 8-bit integer quantization that does not require model retraining
-  or fine-tuning.
+In OpenVINO, the default optimization tool is NNCF (Neural Network Compression Framework).
+It is a `set of compression algorithms <https://github.com/openvinotoolkit/nncf/blob/develop/docs/NNCFArchitecture.md>`__,
```

Comment:
I suggest to refer to NNCF Readme instead.

Reply:
Applied in https://github.com/openvinotoolkit/openvino/pull/25981/commits/853459f89b301b6edb77f8f0c25f2bec6c6371d1